handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -38,6 +39,19 @@
 @Deprecated
 public final class JdkSslServerContext extends JdkSslContext {
 
+    /**
+     * Creates a new instance.
+     *
+     * @param certChainFile an X.509 certificate chain file in PEM format
+     * @param keyFile a PKCS#8 private key file in PEM format
+     * @param keyStore the used keystore type
+     * @deprecated use {@link SslContextBuilder}
+     */
+    @Deprecated
+    public JdkSslServerContext(File certChainFile, File keyFile, String keyStore) throws SSLException {
```

Comment:
It does not make sense to expose a protected method and mark it as @Deprecated from the beginning. Maybe you should remove the annotation and make it package-private ?